src/components/error/FrontEnd/FrontEndErrorContainer.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
```

Comment:
Should be named `FrontEndErrorContainer.react.js` for consistency

Reply:
🙇